content/mooreslaw-tutorial.ipynb

```diff
@@ -0,0 +1,807 @@
+{
```

Comment:
<p>Extra slash here and in the cell below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>